pydsd/DSR.py

```diff
@@ -34,19 +34,27 @@ def tb(D_eq):
        video disdrometer." J. Atmos. Oceanic Technol., 22, 966 - 978. 2005
 
     """
-    if D_eq < 0.7:
-        return 1.0
-    elif D_eq < 1.5:
-        return 1.173 - 0.5165 * D_eq + 0.4698 * D_eq ** 2 - 0.1317 * D_eq ** 3 - 8.5e-3 * D_eq ** 4
+
+    def tb_elementwise(D):
+        """ Quick inner function to map our equation to an array"""
+        if D < 0.7:
+            return 1.0
+        elif D < 1.5:
```

Comment:
Just looked a this again. Will this logic fail if D < 0.7? Given that it meets the first two logic statements. Not sure if it continues through logic or gives up on the first instance of True?

Reply:
It should operate correctly as the elif will only fire if D>=.7.